llvm/lib/Target/AIE/AIEMachineScheduler.cpp

```diff
@@ -1254,6 +1255,8 @@ void AIEScheduleDAGMI::schedule() {
     // If it succeeds, we need to implement it, if we fail we fall back on the
     // normal loop schedule
     SchedImpl->buildGraph(*this, AA);
+    postProcessDAG();
```

Comment:
CHECK : Why do we need the call to "postProcessDAG" ?

Reply:
Some of the DAG mutations are required for correctness, e.g. giving a correct latency to memory edges